mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/jobcluster/JobClusterActor.java

```diff
@@ -613,6 +615,7 @@ MetricGroupId getMetricGroupId(String name) {
 
     @Override
     public void preStart() throws Exception {
+        MDC.put("name", name);
```

Comment:
Is it better to use ThreadLocal to implement it?

Reply:
Oh I got it! I will read the code again and fix it! Thank you for your help!

> what probably needs to happen is some wrapper logger that sets the MDC context before logging a log and then clean it so it doesn' stay in MDC to pollute other logs.

Reply:
> BTW your PR subject/desc doesn't match the change.

It's strange. This seems not the original one that I PR. Because this PR title should match another change that now has been deleted. I will change it or pull a new request later. Thank you!